dotCMS/src/main/java/com/dotmarketing/portlets/templates/action/EditTemplateAction.java

```diff
@@ -494,16 +519,19 @@ else if ((cmd != null)
 	 * @throws DotDataException
 	 * @throws DotSecurityException
 	 */
-	private boolean canTemplateBeDeleted (WebAsset template, User user, StringBuilder errorBuilder)
+	private boolean canTemplateBeDeleted (WebAsset template, User user, SessionDialogMessage errorMessage)
 			throws LanguageException, DotDataException, DotSecurityException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'com.liferay.portal.language.LanguageException', as it cannot be thrown from method's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)